code/run_iLasso_row.m

```diff
@@ -1,55 +1,27 @@
-function [ALasso, metric] = run_iLasso_row(X,params,rownum)
+function [metric] = run_iLasso_row(m,outs,params,rownum)
 dT=params.dT;
 lambda=params.lambda;
 p1=params.p1;
 std_dev=params.kernel_width;
-numrows = max(size(X,1),size(X,2));
-
-if ~(iscell(X))
```

Comment:
Was not previously accessed because we intended to provide the expected cell array, do not need to create a cell array here.